scripts/custom_lint_checks.py

```diff
@@ -375,6 +411,12 @@ def visit_return(self, node):
             )
 
     def visit_yield(self, node):
+        """Called for yield. Checks and adds message regarding yield.
```

Comment:
"Visits a function node that contains a yield statement and verifies that the yield value and the yield type are documented."

Reply:
done.